### ISSUE TITLE
WCAG: Label for file uploader is not read by VoiceOver

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#828]
### Description of the bug

Two issues with our fileuploader

The label is not read by VoiceOver. We should probably test this with NVDA as well.
Wave error: There are more than one label for this component. 
Screenreader:
![Skjermbilde 2023-01-11 kl  13 43 28](https://user-images.githubusercontent.com/33222679/211810332-cc14dfab-89ba-46b3-96f9-2bdf235e3705.png)

More than one label:
![Skjermbilde 2023-01-11 kl  13 47 12](https://user-images.githubusercontent.com/33222679/211810278-2a53f682-d411-465e-b5aa-2752d2ed7e2d.png)

### Steps To Reproduce

Use wave on a form with a file uploader

**Comment by bjosttveit** [comment]
Which label do you mean? This one `Last opp eventuell dokumentasjon for ditt nye navn`?

**Comment by Febakke** [comment]
Yes, if you use Windows you could test it with the NVDA screen reader to see if it is the same there. Seems like I omitted the label for` Last opp eventuell dokumentasjon for ditt nye navn` in the code example but im guessing the multiple label issue is what is causing this. Im not sure, but I think the solution is to make the `Last opp eventuell dokumentasjon for ditt nye navn` the only label and the other texts as `describedby`.

**Comment by bjosttveit** [comment]
I'm using Voice Over, and it currently only reads
```
Dra og slipp eller let etter fil
Tillatte filformater er: .pdf
```

And does not read: `Last opp eventuell dokumentasjon for ditt nye navn`

**Comment by bjosttveit** [comment]
What does `aria-describedby` do? I noticed the container has `file-upload-description file-format-description max-size number-of-attachments` there, and Wave seems to complain about that too

![Image](https://user-images.githubusercontent.com/47412359/214002613-619daee5-b6a8-4047-9fa1-9c5be77ca2c7.png)

**Comment by bjosttveit** [comment]
Fjerna det gamle aria-described-by greiene, for det ser ikke ut som om det gjorde noe annet enn å skape error. Nå har jeg to alternativer som er ganske like, begge fungerer med at alt blir lest opp i riktig rekkefølge, og Wave viser ingen error.

1. Label består kun av `textResourceBindings.title` (Last opp eventuell dokumentasjon for ditt nye navn), og aria-described-by har `textResourceBindings.description`, `Dra og slipp eller let etter fil` og `Tillatte filformater er: .pdf`.
2. Label består av både title og description, og describedBy har kun `Dra og slipp eller let etter fil` og `Tillatte filformater er: .pdf`.

Hva tenker du er best @Febakke ?

**Comment by Febakke** [comment]
Supert! 🙌
Kanskje 1 da det er nærmest resten av komponentene?
